backend/manager/modules/dal/src/main/resources/bundles/AuditLogMessages.properties

```diff
@@ -421,6 +421,8 @@ USER_UPDATE_CLUSTER=Host cluster ${ClusterName} was updated by ${UserName}
 USER_UPDATE_CLUSTER_FAILED=Failed to update Host cluster (User: ${UserName})
 USER_START_CLUSTER_UPGRADE=Successfully set upgrade running flag on Host cluster ${ClusterName} by ${UserName}
 USER_START_CLUSTER_UPGRADE_FAILED=Failed to set upgrade running flag on Host cluster ${ClusterName} by ${UserName}
+USER_UPDATE_CLUSTER_UPGRADE=Successfully updated upgrade percent complete on Host cluster ${ClusterName} by (User: ${UserName})
```

Comment:
It would be nice to include correlation ID and progress value (applies to existing start/finish messages)

Reply:
(will followup)